src/types/apiType.ts

```diff
@@ -0,0 +1,115 @@
+export type ListMovieRequest = {
+    adult: boolean
+    backdrop_path?: string
+    genre_ids: number[]
+    id?: number
+    original_language?: string
+    original_title?: string
+    overview?: string
+    popularity?: Float32Array
+    poster_path?: string
+    release_date?: string
+    title?: string
+    video?: boolean
+    vote_average?: Float32Array
+    vote_count?: number
+}
+
+export type MovieRequest = {
+    adult: boolean
+    backdrop_path: string
+    genres: GenreMovie[]
+    genre_ids: number[] // uniquement pour ElementCard
+    id: number
+    original_language?: string
+    original_title?: string
+    overview: string
+    popularity: Float32Array
+    poster_path: string
+    release_date: string
+    title: string
+    video: boolean
+    vote_average: Float32Array
+    vote_count: number
+}
+
+export type actorCast = {
+    cast: []
```

Comment:
Là vous avez de base indiqué en type `[]` (donc tableau vide), typescript le comprend comme le fait que la valeur sera toujours un tableau vide

```suggestion
    cast: unknown[]
```